tracing-futures/src/tests/mod.rs

```diff
@@ -0,0 +1,22 @@
+#[cfg_attr(
+    not(feature = "std-future"),
+    path = "../../../tracing/tests/support/mod.rs"
+)]
+// We don't need all the test support code that other crates need, so don't
+// warn us if it's unused.
+#[allow(dead_code, unreachable_pub)]
+mod support;
```

Comment:
apparently something in here is breaking rustfmt, may have to figure out a different solution...